app/config/variables.php

```diff
@@ -351,6 +353,14 @@
                 'required' => false,
                 'question' => '',
             ],
+            [
+                'name' => '_APP_FUNCTIONS_ENVS',
+                'description' => 'This option allows you to limit the available environments for cloud functions. This option is very useful for low-cost servers to safe disk space.\n\nTo enable/activate this option, pass a list of allowed environments separated by a comma.\n\nCurrently, supported environments are: ' . \implode(', ', \array_keys(Config::getParam('providers'))),
+                'introduction' => '0.7.0',
+                'default' => '',
```

Comment:
The list is starting to get long, maybe we should think of a default supported list?

Reply:
Which envs would you have in mind?

Reply:
Popularity vs Size formula?

Reply:
Or a Twitter poll 🤣

Reply:
I would go with default values like:

- nodejs (v14)
- deno (v1.6)
- php (v7.4)
- python (v3.8)
- ruby (2.7 or 3.0)
- .NET (v5.0)

Or still too long?

Reply:
LGTM